framework/test/mocha/functional/http/get/accounts/accounts.js

```diff
@@ -304,7 +305,10 @@ describe('GET /accounts', () => {
 				return accountsEndpoint.makeRequest({}, 200).then(res => {
 					const balances = _.clone(res.body.data);
 					expect(
-						_.orderBy(balances, ['balance', 'address'], ['asc', 'asc'])
+						balances.sort((a, b) => {
+							const aBignumBalance = new Bignum(a.balance);
+							return aBignumBalance.minus(b.balance) || a.address - b.address;
```

Comment:
```suggestion
							return aBignumBalance.minus(b.balance) || (a.address > b.address) ? 1 : -1;
```

Reply:
above the comparison might be the opposite for address

Reply:
I used `localeCompare` to be safe in the end.